R/utils-classes.R

```diff
@@ -113,6 +113,14 @@ print.bcdc_recordlist <- function(x, ...) {
   cat("\nTitles:\n")
   x <- purrr::set_names(x, NULL)
   cat(paste(purrr::imap(x[1:n_print], ~ {
+
+    if (nrow(x[[.y]][["resource_df"]]) == 0) {
```

Comment:
can we use `bcdc_tidy_resources` here instead?
```suggestion
    if (!nrow(bcdc_tidy_resources(x[[.y]]))) {
```